packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -2258,16 +1988,9 @@ describe( 'Controlling link title text', () => {
 	const selectedLink = fauxEntitySuggestions[ 0 ];
 
 	it( 'should not show a means to alter the link title text by default', async () => {
-		act( () => {
-			render(
-				<LinkControl value={ selectedLink } forceIsEditingLink />,
-				container
-			);
-		} );
+		render( <LinkControl value={ selectedLink } forceIsEditingLink /> );
 
-		expect(
-			queryByRole( container, 'textbox', { name: 'Text' } )
-		).toBeFalsy();
+		expect( screen.queryByRole( 'textbox', { name: 'Text' } ) ).toBeFalsy();
```

Comment:
Is this also meant to be `.not.toBeInTheDocument()`?

```suggestion
		expect( screen.queryByRole( 'textbox', { name: 'Text' } ) ).not.toBeInTheDocument();
```